rpc/block.go

```diff
@@ -8,24 +8,24 @@ import (
 )
 
 // https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_api_openrpc.json#L1999-L2008
-type BlockStatus uint8
+type Status uint8
 
 const (
-	BlockStatusPending BlockStatus = iota
-	BlockStatusAcceptedL2
-	BlockStatusAcceptedL1
-	BlockStatusRejected
+	StatusPending Status = iota
+	StatusAcceptedL2
+	StatusAcceptedL1
+	StatusRejected
```

Comment:
Maybe this?
```suggestion
	Pending Status = iota
	AcceptedOnL2
	AcceptedOnL1
	Rejected
```

Reply:
With the prefix I find it easier to enumerate all possible "Status" values while actually coding.
after typing only "Status" goland suggests you all the possible values, which I find to be neat:)

also prefix avoids any conflicts. another enum type could have a Pending value as well.